src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAdminController.java

```diff
@@ -146,6 +150,12 @@ public void fullDeleteStudy(Long[] ids) {
         return users.map(user -> conversionService.convert(user, CommonUserDTO.class));
     }
 
+    @ApiOperation("Register new users")
+    @RequestMapping(value = "/api/v1/admin/users/registration", method = RequestMethod.POST)
```

Comment:
Why not REST convention (POST "api/v1/admin/users")?

Reply:
because it will lead to ambiguous mapping. We have (POST "api/v1/admin/users") endpoint in the BaseUserController

Reply:
Then this one is smth like "api/v1/admin/users/batch"